test/e2e/General/General.js

```diff
@@ -0,0 +1,22 @@
+/* global browser */
+/* global protractor */
+/* global element */
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["element" is defined but never used](https://www.codacy.com/app/MitocGroup/deep-microservices-todo-app/file/2096183354/issues/source?bid=2875840&fileBranchId=2992404#l3)